### PR TITLE
Implement desktop version

### PR DESCRIPTION
**Pull request by DjibrilM**
### In the following pull request I added 
- responsiveness for desktop screens

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -390,7 +463,7 @@ <h2 class="contact-title">Contact me</h2>
                         placeholder="Write your message here..." type="text"></textarea>
                 </div>
 
-                <div class="">
+                <div class="contact-form-button-container">
```
Comment (usorfaitheloho) [inline index.html:466]: Please use the right background color for the `hover` interaction of the `Get in touch` button of the `contact me` section to help you with a better UI : 

Button interaction :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
## Hi @DjibrilM   :clap:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight :clap:  
:arrow_forward: Followed the Figma design template :heavy_check_mark: 
:arrow_forward: Good code quality :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — COMMENTED** [review]
Hi @DjibrilM 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that the GitHub actions can't carry out linters checks because of branch conflict. Kindly resolve the branch conflict and ensure that the linters actions run 👍

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -1,569 +1,476 @@
...
-        </a>
-      </div>
-
-      <div class="tools-summery">
-        <div class="header">
-          <h2>languages</h2>
-          <i class="bx bx-chevron-down"></i>
+        <div class="about-infos">
+            <div class="info-description">
+                <h2>About Myself</h2>
```
Comment (wickathou) [inline index.html:316]: - [ ] **Required:** As my colleague mentioned, your 'About Myself' font color and size must be:
`color: #091E42;`
`font-size: 40px;`

File: index.html [hunk 3]
```diff
@@ -1,569 +1,476 @@
...
+
+            <div class="contact">
+                <a href="">Get my resume</a>
+            </div>
+
+            <div class="showcase-contact-icons">
+                <a href="">
+                    <svg class="showcase-icon" width="20" height="20" viewBox="0 0 20 20" fill="currentColor"
+                        xmlns="http://www.w3.org/2000/svg">
+                        <path
```
Comment (wickathou) [inline index.html:332]: - [ ] **Required:** As mentioned by my colleague, your 'About Myself' section is not arranged as shown on the figma file, as your are missing the 'LET's CONNECT' anchor tag and your `showcase-contact-icons` should be above the 'Get my resume' button

File: index.html [hunk 4]
```diff
@@ -1,569 +1,476 @@
...
+                <div class="tool-dropdown">
+                    <p>Frameworks</p>
+                    <i class='bx bx-chevron-right'></i>
+                </div>
+
+                <div class="tool-dropdown">
+                    <p>Frameworks</p>
+                    <i class='bx bx-chevron-right'></i>
+                </div>
+            </div>
```
Comment (wickathou) [inline index.html:406]: - [ ] **Required:** You have the 'Frameworks' text repeated, when it must be 'Frameworks' and 'Skills', please update it as my colleague also mentioned

Comment (wickathou) [inline index.html:406]: [Optional] The `tools-summery` section should be a `ul` listing the 'Languagues', 'Frameworks' and 'Skills' with `li` elements which then contain their own `ul` and `li` elements to present their own lists

**Review by wickathou — CHANGES_REQUESTED** [review]
Hi @DjibrilM  , 

## Great work on making the changes requested by a previous reviewer

# Just a few changes more and you will be done!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — APPROVED** [review]
Hi @DjibrilM , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You implemented changes requested by previous reviewers very well 
🚀 You have descriptive commit messages and you constructed them with imperative statements
🚀 You used GitHub flow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**